pkg/kubelet/kubelet.go

```diff
@@ -1966,10 +1967,12 @@ func (kl *Kubelet) canAdmitPod(pods []*v1.Pod, pod *v1.Pod) (bool, string, strin
 	attrs := &lifecycle.PodAdmitAttributes{Pod: pod, OtherPods: pods}
 	for _, podAdmitHandler := range kl.admitHandlers {
 		if result := podAdmitHandler.Admit(attrs); !result.Admit {
+			klog.InfoS("Pod is rejected", "admitHandler", reflect.TypeOf(podAdmitHandler), "pod", klog.KObj(pod), "podUID", pod.UID, "result", result)
```

Comment:
I'm not completely sure we want to use `reflect` here, is it used elsewhere in the code/package?